packages/rn-tester/package.json

```diff
@@ -9,6 +9,11 @@
     "url": "git@github.com:facebook/react-native.git",
     "directory": "packages/rn-tester"
   },
+  "scripts": {
+    "jsc": "bundle install && bundle exec pod install",
+    "hermes": "bundle install && USE_HERMES=1 bundle exec pod install",
+    "clean": "rm -rf build/generated/ios && rm -rf Pods && rm Podfile.lock"
```

Comment:
Those two commands are "ios-centric". They should be adapted to work with both Android & iOS. Plus `yarn jsc` feels odd as a command. It should rather be `yarn setup-jsc` or something similar

Reply:
Thank you for your suggestions!

I added Android commands too. Please take a look.